Add stock option grant timing policy (#1649)
sourcegraph/handbook: content/benefits-pay-perks/pay-expenses/compensation/equity-timing.md

# How and when stock option grants are calculated

We calculate option grants for all teammates (at new hires, promotions, refreshes, etc.) using a simple calculator that takes three key inputs:

- The **gross equity value**, in dollars, that they will receive (based on our [compensation bands](index.md#compensation-bands)).
- The **common stock fair market value** (“common FMV”), which becomes the “strike price” or “exercise price” that they have to pay to exercise their options.
- The **preferred stock price**, or the latest amount that investors paid for Sourcegraph preferred shares.

## Why timing matters

All three of the key inputs above can change at any time. In some cases, we have visibility of those changes—e.g., if we have a plan to update compensation bands. In many cases, however, we do not—e.g., if we receive a term sheet from an investor, suddenly our common FMV would change. If we complete a fundraising round, our preferred stock price would change.

Changes to any of these inputs can affect the size and value of stock option grants. This is one reason why all offers note that the calculation of stock options is illustrative only.

## Principle

> The principle behind this policy is that teammates should benefit from (or be affected by) increases or changes in valuation that occur once they have joined the company or are performing their new duties in a new position. If our valuation increases before somebody joins, for example, their option grant would be adjusted using the methodology below to reflect the new stock price.

Author: ```suggestion
The principle behind this policy is that teammates should benefit from (or be affected by) increases or changes in valuation that occur once they have joined the company or are performing their new duties in a new position. 

On the other hand, if our valuation increases or changes before somebody joins, their option grant would be adjusted using the methodology below to reflect the new stock price.
```

## Option grant timing

For any stock option grant...

### Common FMV

The applicable common FMV is determined on the date that Sourcegraph's board of directors approves the option grant (this is legally required).

For new hires, this cannot happen until after they start working at Sourcegraph.

Common FMVs change periodically, based on several potential events. As an example, companies must update their common FMV at least once per year. As another example, if Sourcegraph receives a term sheet from a potential investor, the United States IRS will likely take the position that the price of the stock has changed, and the common FMV must be updated as well. Similarly, if Sourcegraph receives word that an investor sold their stock to another party at a new price, the same thing may happen. We often don’t have full control over if and when these “material events” happen (such as when an investor will send a term sheet).

We strive to get option grants approved prior to any known or expected changes in common FMV.

### Gross equity value

> The gross equity value granted to somebody will be determined from the prevailing compensation bands on the date that an offer is presented to a candidate or the date that a promotion is communicated to a teammate.

Author: ```suggestion
The gross equity value granted to somebody will be determined from the prevailing [compensation bands](index.md#compensation-bands) on the date that an offer is presented to a candidate or the date that a promotion is communicated to a teammate.
```

### Preferred stock price

> The preferred stock price used for the calculation of the number of options will be the "prevailing" preferred stock price on the date that someone starts or the effective date of a promotion.

Author: ```suggestion
The preferred stock price used for the calculation of the number of options will be the "prevailing" preferred stock price on the date that someone starts or the effective date of a promotion or refresh.
```

If a stock transaction (e.g. a fundraising) occurs, prevailing preferred stock prices will be updated at the same time that common FMVs are updated (generally: when we have reasonable certainty that a transaction will occur or that an offered price is legitimate). For example, if we receive a term sheet from an investor that we believe reflects legitimate interest, that would likely trigger changes to both common FMV (as described above) and in our prevailing preferred stock price for option grant calculations.

## Examples

Each of the examples below align with the [principle](#principle) behind this policy above.

### Example 1: Logan's new hire grant

> - Logan signs an offer letter to join Sourcegraph on January 1 to receive $100 of gross equity value. The common FMV at that point in time is $1, and the prevailing preferred stock price is $2. Their offer summary calculator shows an illustrative grant of 50 options ($100 divided by $2), each with a strike price of $1. They accept, and selects a start date of March 1.

Author: ```suggestion
- Logan signs an offer letter to join Sourcegraph on January 1 to receive $100 of gross equity value. The common FMV at that point in time is $1, and the prevailing preferred stock price is $2. Their offer summary calculator shows an illustrative grant of 50 options ($100 divided by $2), each with a strike price of $1. They accept, and select a start date of March 1.
```

- Sourcegraph receives a term sheet from a new investor on February 1 at a new, higher valuation (a $4 preferred stock price). Given the investor's legitimacy and familiarity with Sourcegraph and the industry, our valuation advisors determine that Sourcegraph's common FMV should be updated to $2 to reflect this new pricing information.

> - Logan starts working at Sourcegraph on March 1. Their gross equity value received remains $100 (it was locked in when she signed her offer letter). They receive 25 options ($100 divided by $4), each with a strike price of $2.

Author: ```suggestion
- Logan starts working at Sourcegraph on March 1. Their gross equity value received remains $100 (it was locked in when they signed their offer letter). However, they receive 25 options ($100 divided by $4), each with a strike price of $2.
```

### Example 2: Jordan's new hire grant

- Similar to Logan, Jordan signs an offer letter on January 1 to receive $100 of gross equity value. However, Jordan selects a start date of January 20.
- Jordan starts at Sourcegraph on January 20, and the board of directors approves their stock option grant the following week. Their gross equity value received remains $100, and they receive 50 options (using the still prevailing preferred stock price of $2) with a strike price of $1 (using the current common FMV).

### Example 3: Morgan's promotion

> - Morgan is notified that they are being promoted on February 15, and it is effective on the same day. This occurs after the fundraising described in example 1 above. They receive a stock option grant worth $32 of gross equity value. The common FMV at that point in time is $2, and the prevailing preferred stock price is $4. Their offer summary calculator shows an illustrative grant of 8 options ($32 divided by $4), each with a strike price of $2.

Author: ```suggestion
- Morgan is notified that they are being promoted on February 15, and it is effective on the same day. This promotion becomes effective _after_ the term sheet described in example 1 above is received from an investor. They receive a stock option grant worth $32 of gross equity value. The common FMV at that point in time is $2, and the prevailing preferred stock price is $4. Their offer summary calculator shows an illustrative grant of 8 options ($32 divided by $4), each with a strike price of $2.
```
